tests/longevity/longevity.md

```diff
@@ -0,0 +1,149 @@
+# Longevity Test
+
+This document describes how we test NGF for longevity.
+
+<!-- TOC -->
+
+- [Longevity Test](#longevity-test)
+  - [Goals](#goals)
+  - [Test Environment](#test-environment)
+  - [Steps](#steps)
+    - [Start](#start)
+    - [Check the Test is Running Correctly](#check-the-test-is-running-correctly)
+    - [End](#end)
+  - [Analyze](#analyze)
+  - [Results](#results)
+
+<!-- TOC -->
+
+## Goals
+
+- Ensure that NGF successfully processes both control plane and data plane transactions over a period of time much
+  greater than in our other tests.
+- Catch bugs that could only appear over a period of time (like resource leaks).
+
+## Test Environment
+
+- A Kubernetes cluster with 3 nodes on GKE
+  - Node: e2-medium (2 vCPU, 4GB memory)
+  - Enabled GKE logging.
+  - Enabled GKE Cloud monitoring with managed Prometheus service, with enabled:
+    - system.
+    - kube state - pods, deployments.
+- Tester VMs:
```

Comment:
Where do these VMs exist?

Reply:
Maybe belongs in an internal doc to explain how a dev sets all of these things up in GKE.

Reply:
I think it will repeat Google Cloud documentation on how to create a VM. The important parts (reflected in the text) are that it needs to be in the same zone , it should be Debian and tmux and wrk needs to be installed